content/news/2024-09-25-bevy-foundation-501c3/index.md

```diff
@@ -0,0 +1,65 @@
++++
+title = "Bevy Foundation is now a 501(c)(3) Public Charity"
+date = 2024-09-25
+authors = ["Carter Anderson"]
+[extra]
+twitter = "cart_cart"
+github = "cart"
+youtube = "cartdev"
++++
+
+In March we [announced the Bevy Foundation](/news/bevy-foundation/), our new non-profit dedicated to developing Bevy and teaching people how to use it. At that time, we applied for federal 501(c)(3) public charity status, which would give us tax-exempt status in the United States and make donations tax-deductible. It introduces stricter operational constraints that help protect the integrity of our mission and our community.
+
+I am excited to announce that the Bevy Foundation has been granted 501(c)(3) public charity status!
+
+<!-- more -->
+
+## What does this mean for Bevy?
+
+1. We are exempt from income tax on donations we receive (including the donations we have already received this year). This means 100% of the donations we receive go toward furthering our mission.
+2. Donations to Bevy Foundation (including all past donations that occurred this year) are now tax-deductible in the United States.
+3. Many employers have donation-matching programs for 501(c)(3)s. It is worth checking to see if your employer will match your donation!
+4. There are many grant opportunities out there for 501(c)(3)s. These are now on the table for the Bevy Foundation.
+5. We now must [operate under the constraints of a 501(c)(3) public charity](https://www.501c3.org/what-is-a-501c3/)
+6. We must file a yearly report that outlines our activities (IRS form 990).
+
+I am personally very excited about this. 501(c)(3) public charity status, because of its constraints, buys us more public trust than the more general state level non-profit status. Each donation can go further, and there are now more opportunities for financial support!
```

Comment:
```suggestion
I am personally very excited about this. 501(c)(3) public charity status, because of its constraints, buys us more public trust than the more general state-level non-profit status. Each donation can go further, and there are now more opportunities for financial support!
```